### ISSUE TITLE
mass.off.channels.getAttr should take a states argument

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#184]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/184) by Galen O'Neil (Bitbucket: [oneilg](https://bitbucket.org/oneilg), GitHub: [oneilg](https://github.com/oneilg)).**

----------------------------------------

Almost all uses look like this

```python
        inds = self.getStatesIndicies(states)
        indicator, uncorrected = self.getAttr([indicatorName, uncorrectedName], inds, goodFunc, returnBad)
```

so it seeems like that could be reduced to one function call quite easily

**Comment by joefowler** [comment]
**Original comment by Galen O'Neil (Bitbucket: [oneilg](https://bitbucket.org/oneilg), GitHub: [oneilg](https://github.com/oneilg)).**

----------------------------------------

fixed by PR #119

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Removing milestone: v0.7 (automated comment)
